lib/config.ts

```diff
@@ -69,3 +74,17 @@ export function findTsConfigFiles() {
   const tsConfigs = glob.sync("**/*tsconfig*.json");
   return tsConfigs;
 }
+
+export function getGitHubTags(): string[] {
+  return execSync("git tag").toString().trim().split("\n").reverse();
+}
+
+export function getGitHubBranches(defaultTag: string): string[] {
+  return [
+    defaultTag,
+    ...execSync("git branch").toString().trim().split("\n")
+      // @TODO remove ugly hack for the `*` char of the current branch
+      .map(i => i.split("* ").join(""))
+      .filter(v => v === defaultTag)
```

Comment:
I haven;t tried it locally, but won't `getGitHubBranches` return `[master, master]`?

```suggestion
      .filter(v => v !== defaultTag)
```